### DISCUSSION TITLE
License locked (revalidating)

### DISCUSSION THREAD

**Discussion by MGadAllah** [aclap-dev/video-downloadhelper#355] [Help and Q&A]
### Your question

Hi,
Why it shows:
"License locked (revalidating)"

### Addon & CoApp version and details

License locked (revalidating)

**Comment by jcfrog** [comment]
Hi,
You should have received an email to unlock the license, please check your spams.

**Reply by guitarflipper** [reply]
My revalidating email never came. I have Firefox and I first got that message in the settings about two weeks ago. I have two primary computers that I use this for, a desktop and a new laptop (that gets the error).

**Reply by jcfrog** [reply]
please send me your email address at licenses@downloadhelper.net
